plugins/degu.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Plugins for using Degu elements.
+ */
+
+import {NunjucksTemplateEngine, Pod} from '@amagaki/amagaki';
+
+import nunjucks from 'nunjucks';
+
+export class DeguPlugin {
+  static register(pod: Pod) {
+    const nunjucksEngine = pod.engines.getEngineByExtension(
+      '.njk'
+    ) as NunjucksTemplateEngine;
+    nunjucksEngine.env.addGlobal('degu', {
+      asset: DeguPlugin.asset,
+    });
+  }
+
+  static asset(options: {
```

Comment:
I wonder if it is better to have `degu-asset` web-component that would be a composition of `degu-image` and `degu-video`.   I feel like from the perspective of degu as a library, that provides more flexibility across usage / platforms but I'm okay either way.

Reply:
Yeah, I think generally across our projects we want to support video or images anywhere an image is used, so I'd be supportive of this. That said, there are attributes that only apply to videos (and potentially vice versa) so it might start to get a little overloaded. It'll be good to work through that though.